src/Pages/License/Search/SearchLicenseDetails.jsx

```diff
@@ -0,0 +1,99 @@
+import Grid from "@mui/material/Grid";
+import Stack from "@mui/material/Stack";
+import Divider from "@mui/material/Divider";
+import Typography from "@mui/material/Typography";
+import CustomCard from "Components/CustomCard/CustomCard";
+import CustomCardContent from "Components/CustomCard/CustomCardContent";
+import CustomChip from "Components/CustomCard/CustomChip";
+import ProductKeyInformationCard from "./ProductKeyInformationCard";
+
+const SearchLicenseDetails = () => {
+  return (
+    <Grid sx={{ pl: 5}}>
+      <CustomCard style={{ width: "37vw", height: "22vw" }}>
+        <CustomCardContent>
+          <Grid item>
+            <Grid item container xs alignItems="center">
+              <Typography variant="body1">Overview</Typography>
+            </Grid>
+            <Divider sx={{ borderBottomWidth: 2, m: 2 }} />
+            <Grid item sx={{ m: 1 }}>
+              <Stack spacing={1} direction="row">
+                <CustomChip
+                  label={"Product Key"}
+                  value="1x83-5684-c832-1b5744"
+                  chipWidth={"200px"}
+                ></CustomChip>
+                <CustomChip
+                  label={"Feature ID"}
+                  value="CC_PHOPT"
+                  chipWidth={"200px"}
```

Comment:
string can be passed without using curly braces

Reply:
done